pkg/kubelet/kubelet.go

```diff
@@ -836,7 +836,14 @@ func NewMainKubelet(kubeCfg *kubeletconfiginternal.KubeletConfiguration,
 		util.SetNodeOwnerFunc(klet.heartbeatClient, string(klet.nodeName)))
 
 	// setup node shutdown manager
-	shutdownManager, shutdownAdmitHandler := nodeshutdown.NewManager(klet.GetActivePods, killPodNow(klet.podWorkers, kubeDeps.Recorder), klet.syncNodeStatus, kubeCfg.ShutdownGracePeriod.Duration, kubeCfg.ShutdownGracePeriodCriticalPods.Duration)
+	shutdownManager, shutdownAdmitHandler := nodeshutdown.NewManager(&nodeshutdown.Config{
```

Comment:
We should add a log message somewhere, maybe here, that logs the various parameters for the shutdown manager. The durations and PodPriorityShutdownGracePeriod would be extremely helpful.

Reply:
+1

Reply:
https://github.com/kubernetes/kubernetes/blob/7366ed641bac85c1352e354410427deab9a920be/pkg/kubelet/kubelet.go#L839

Done